allensdk/brain_observatory/behavior/behavior_session.py

```diff
@@ -1135,7 +1141,42 @@ def stimulus_templates(self) -> pd.DataFrame:
                     image array of warped stimulus image
 
         """
-        return self._stimuli.templates.value.to_dataframe()
+        if self._stimuli.templates.image_template_key is not None:
+            return self._stimuli.templates.value[
+                self._stimuli.templates.image_template_key
+            ].to_dataframe()
+        else:
+            return None
+
+    @property
+    def stimulus_template_movie(self) -> pd.DataFrame:
```

Comment:
this is specifically returning the fingerprint movie stimulus template. please name it that so that we can easily support adding another movie type in the future if needed